### PR TITLE
GA CI workflow

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ jobs:
         fi
 
     - name: Submit code coverage
-      if: success() && matrix.php-version == '7.4'
```
Comment (othercorey) [inline .github/workflows/ci.yml:121]: I don't have a problem with these changes, but wondering what this fixes.

Comment (othercorey) [reply .github/workflows/ci.yml:121]: Is it so the patch codecov % isn't way off for a single failure?

Comment (othercorey) [reply .github/workflows/ci.yml:121]: Now I'm thinking the success() is just redundant.

Comment (ADmad) [reply .github/workflows/ci.yml:121]: > Now I'm thinking the success() is just redundant.

Yes, as stated in the commit message.

**Pull request merged**